### PR TITLE
Fix an incorrect usage of strip which caused bad formats to be read

### PR DESCRIPTION
**Pull request by akariv**

### CODE REVIEW

**Comment by zhenyab** [comment]
@akariv could you please explain the use case?

**Comment by akariv** [comment]
This is a simple bug:

a.strip("fmt:") removes all instances of 'f','m','t,':' from the beginning and the end of the string.

e.g. 'fmt:%y-%m' will be converted to '%y-%' which is not what you'd want here.

**Comment by roll** [comment]
@akariv 
Thanks! Will you bump version in PR for quick release?

**Comment by vitorbaptista** [comment]
@akariv Interesting catch. Changing `strip('fmt:')` to `lstrip('fmt:')` seems to solve it as well, while keeping the code a bit more explicit than `[4:]`.

**Comment by akariv** [comment]
@vitorbaptista not necessarily as 'fmt:my-format' will still be decoded incorrectly.
@roll done

File: jsontableschema/types/base.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def __init__(self, field=None):
         self.__format_fmt = None
         if self.__format.startswith('fmt'):
```
Comment (akariv) [inline jsontableschema/types/base.py]: True. Fixed.

**Comment by pwalsh** [comment]
more to the point, we need to remove `fmt:` from the spec: https://github.com/frictionlessdata/specs/issues/260

**Comment by roll** [comment]
@pwalsh 
I know but we're waiting spec for now to make changes here right?

**Comment by roll** [comment]
@akariv 
released as 0.6.5 - https://pypi.python.org/pypi/jsontableschema

**Pull request merged**